src/OrchardCore/OrchardCore.Users.Core/Authentication/CacheTicketStore.cs

```diff
@@ -0,0 +1,166 @@
+using System;
+using System.IO;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace OrchardCore.Users.Authentication;
+
+public class CacheTicketStore : ITicketStore
+{
+    private const string _keyPrefix = "ocauth-ticket";
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private IDataProtector _dataProtector;
+
+    public CacheTicketStore(IHttpContextAccessor httpContextAccessor)
+    {
+        _httpContextAccessor = httpContextAccessor;
+    }
+
+    public async Task RemoveAsync(string key)
+    {
+        var cacheKey = String.Concat(_keyPrefix, "-", key);
```

Comment:
We proved that the interpolated string was actually doing this, hence you could keep it ;)

Reply:
:) reverted to use interpolated string